test/functional/feature_segwit.py

```diff
@@ -212,6 +207,7 @@ def run_test(self):
         assert tmpl['sigoplimit'] == 80000
         assert tmpl['transactions'][0]['txid'] == txid
         assert tmpl['transactions'][0]['sigops'] == 8
+        assert '!segwit' in tmpl['rules']
```

Comment:
shouldn't we also test for `'csv'` being present here?

Reply:
Going to merge this to not invalidate ACKs. Test improvements are welcome and can be submitted in a follow-up